### ISSUE TITLE
[naming-convention] Add Query-specific option

### ISSUE THREAD

**Issue by eddeee888** [dotansimha/graphql-eslint#253]
I think there could be a new option that targets the queries because the naming convention could be different from a field definition on a normal type. 

For example, I may want to forbid the queries to have `get` prefixes because queries are always "GET" but still allows other object type definitions, then I can add the following rule to eslint config:

```json
"@graphql-eslint/naming-convention": [
  "error",
  {
    "QueryFieldDefinition": { "format": "camelCase", "forbiddenPrefixes": ["get"] },
    "FieldDefinition": { "format": "camelCase" },
  }
]
```

```graphql
type Query {
  getUser: User # Error
  user: User # OK
}

type User {
  getter: User # OK
}
```

If it makes sense I can try creating a PR for this :)

**Comment by dotansimha** [comment]
We have this: https://github.com/dotansimha/graphql-eslint/blob/master/docs/rules/avoid-operation-name-prefix.md :) 

```
@graphql-eslint/avoid-operation-name-prefix: ["error", [{"keywords":["get"]}]]
```

**Comment by eddeee888** [comment]
I had a look and I think [avoid-operation-name-prefix](https://github.com/dotansimha/graphql-eslint/blob/d9e93779ea76fa112af0f5d2fb21324b155a209d/packages/plugin/src/rules/avoid-operation-name-prefix.ts#L103-L108) only works for `OperationDefinition` and `FragmentDefinition` at the moment? 
I think it would be nice to be able to enforce a similar rule to query definitions in the schema files as well :)

**Comment by dotansimha** [comment]
Oh, that totally makes sense, mixed the two :) Re-opening :)

**Comment by eddeee888** [comment]
Awesome thanks! I should have made it clearer that it's for schema files. I'll look into this and create a PR soon :)

**Comment by eddeee888** [comment]
@dotansimha I have created a PR for this: https://github.com/dotansimha/graphql-eslint/pull/256
Please have a look when you have a chance, thanks! 🙏

**Comment by dotansimha** [comment]
Available in `@graphql-eslint/eslint-plugin@0.8.0`
